demo/api/src/config/environment-variables.ts

```diff
@@ -142,4 +142,8 @@ export class EnvironmentVariables {
 
     @IsString()
     CAMPAIGN_BASIC_AUTH_PASSWORD: string;
+
+    @IsNumber()
+    @Type(() => Number)
+    BREVO_FOLDER_ID: number;
```

Comment:
This should be optional. Else this is a breaking change. Also making it required would not allow overwriting the default but would just be a required config and the default value would never be used

Reply:
Fixed here: 65030841040a19cfdb787abdc949c32f8e0eea01